src/views/Home/Article/ArticleDetail/index.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useState, useCallback } from 'react';
+
+import {
+    isDefined,
+    _cs,
+} from '@togglecorp/fujs';
+
+import {
+    IoClose,
+    IoCheckmarkSharp,
+} from 'react-icons/io5';
+
+import {
+    Modal,
+    Button,
+    PendingMessage,
+    Tabs,
+    TabList,
+    Tab,
+    TabPanel,
+} from '@togglecorp/toggle-ui';
+
+import TagInput from '#components/TagInput';
+
+import { useRequest } from '#utils/request';
+
+import styles from './styles.css';
+
+interface ArticleDetail {
+    title: string;
+    url: string;
+    articleId: string;
+    content: string;
+    date: string;
+    language?: string;
+    locations?: string[];
+    regions?: string[];
+    tags?: string[];
+    themes?: string[];
+    searchScore: number;
+}
+
+interface Props {
+    onModalClose: () => void;
+    articleId: string;
+    onMarkAsRelevantClick: () => void;
+    onMarkAsIrrelevantClick: () => void;
+}
+
+function ArticleDetail(props: Props) {
+    const {
+        onModalClose,
+        articleId,
+        onMarkAsRelevantClick,
+        onMarkAsIrrelevantClick,
+    } = props;
+
+    const [tabsValue, setTabsValue] = useState('extracted-text');
```

Comment:
Define possible tabs as type and assign it here.
```suggestion
    type TabVariant = 'extracted-text' | 'smth';
    const [tabsValue, setTabsValue] = useState<TabVariant>('extracted-text');
```